src/routes/redirector.ts

```diff
@@ -9,10 +9,21 @@ export default async function (fastify: FastifyInstance): Promise<void> {
     '/:postId',
     async (req, res) => {
       const con = await createOrGetConnection();
-      const post = await con.getRepository(ArticlePost).findOne({
-        select: ['id', 'url', 'tagsStr'],
-        where: [{ id: req.params.postId }, { shortId: req.params.postId }],
-      });
+      const q = con
+        .createQueryBuilder()
+        .select([
+          'post.id AS id',
+          'post.url AS url',
+          'post.tagsStr AS "tagsStr"',
+        ])
+        .from('post', 'post')
```

Comment:
Why don't we use `Post, 'post'` notation here?
I think with that the select can also be cleaned.

Reply:
Just `Post` is enough I think as well.

Reply:
Ah, yes, we can. In earlier tests it didn't work because it would then add the WHERE type, or complain about url property not existing

Reply:
Changed it to `Post, 'post'`. But also without `AS` it returns them as post_id etc, so I added `AS` to "throw" them back to their correct name.